requests_kerberos/kerberos_.py

```diff
@@ -79,14 +79,14 @@ def _negotiate_value(response):
     return None
 
 
-class HTTPKerberosAuth(AuthBase):
+class HTTPGSSAPIAuthBase(AuthBase):
```

Comment:
Do we intend for users to wrap this base-class further? If not, maybe we should prefix it with an `_` to make it easier to change in the future without breaking peoples code?